ds_judgements_public_ui/templates/pages/home.html

```diff
@@ -73,7 +73,7 @@ <h1 class="service-introduction__header-beta">Find case law</h1>
         <div class="service-introduction">
           <span class="service-introduction__separator"></span>
           <h1 class="service-introduction__header">Find case law</h1>
-          <p class="service-introduction__helper-text">{% translate "home.useservice" %}</p>
```

Comment:
More a question than a comment - why are we translating anything? Is there a plan to make the site have more than 1 language?

Reply:
Long version -- https://github.com/nationalarchives/ds-find-caselaw-docs/blob/main/doc/adr/0015-translations.md
Short version -- we did it early on and then decided it was a bad idea, so whilst we haven't committed time to removing all the translations, we're generally in favour of swapping it out for plain text.

Welsh language UI is not on the roadmap. It's not impossible that we get a dual Welsh/English judgment at some point -- we might just wind up concatenating them.